### ISSUE TITLE
Components that take input are not draggable

### ISSUE THREAD

**Issue by DanielGelfand** [daybrush/moveable#479]
## Environments
* Framework name: React
* Framework version: 17.0.2
* Moveable Component version: 0.27.7
* Testable Address(optional):

## Description
I am having trouble dragging and resizing components which take input or are clickable (buttons). Input components do not drag and button components move very slowly when being dragged. The screen recording below displays the issue.

https://share.getcloudapp.com/P8u901LO

**Comment by daybrush** [comment]
@DanielGelfand 

Isn't it a transition to the button?

**Comment by DanielGelfand** [comment]
No, I am able to drag any of the 3 type of components into my iframe. However, only the Card is fully functional.

**Comment by daybrush** [comment]
@DanielGelfand 

Do you have any test code? The only thing I can infer from the video is css transition.

**Comment by DanielGelfand** [comment]
https://codesandbox.io/s/awesome-dust-ssk12

Seems like the problem is due to antd styling.

**Comment by daybrush** [comment]
@DanielGelfand 
ant-input has transitions.
```css
.ant-input {
    transition: all 0.3s;
}
```
